src/screens/MapScreen.tsx

```diff
@@ -10,17 +10,30 @@ import { poiApi } from "@api/client/apis";
 import { GetAllPOI200ResponseInner } from "@api/generated/api";
 import { displayError } from "@utils/error";
 import { AuthContext } from "@contexts/auth";
+import { ThemeContext } from "@contexts/theme";
 import { userGeolocationState } from "@atoms/geolocationAtom";
 import { StyledText } from "@components/StyledText";
+import * as STYLES from "@constants/styles";
```

Comment:
If all you're using is `WHITE_COLOR` you can do `import { WHITE_COLOR }`

Reply:
Good catch, was in a rush and didn't realize

Reply:
It's not necessarily bad, infact it probably could be argued that it could help readability if you don't really care about memory for importing more things than you need.